app/helpers/admin_helper.rb

```diff
@@ -103,4 +103,11 @@ def admin_participant_role_name(class_name)
       "unknown"
     end
   end
+
+  def training_statuses
+    [
+      OpenStruct.new(id: "deferred", value: "deferred"),
+      OpenStruct.new(id: "withdrawn", value: "withdrawn"),
```

Comment:
Can we capitalise these in the radio button labels?

```suggestion
      OpenStruct.new(id: "deferred", value: "Deferred"),
      OpenStruct.new(id: "withdrawn", value: "Withdrawn"),
```